ui/components/MesheryPatterns.js

```diff
@@ -145,6 +147,12 @@ const YamlDialogTitleText = styled(Typography)(() => ({
   flexGrow: 1,
 }));
 
+const DesignFilterWrapper = styled(Box)(() => ({
```

Comment:
don't put styles over same file, there should style file where all other styles of this page would have defined. This clutter the things in same file.